modules/ocf_desktop/files/xsession/lab_close_notify

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import datetime
+import sched
+import subprocess
+import time
+
+from ocflib.lab import hours
+
+# 0 minutes, 2 minute, 5 minutes, 10 minutes, 15 minutes
+WARNING_DURATIONS = [0, 120, 300, 600, 900]
+
+def notify_user(seconds_left):
+    title = 'Lab closure'
+    if seconds_left != 0:
+        msg = 'Lab closing in {0:g} minutes'.format(seconds_left / 60)
+    else:
+        msg = 'Lab is closing NOW!'
+
+    # TODO: pick an icon
+    subprocess.call(['notify-send', title, msg])
+
+today = datetime.datetime.today()
```

Comment:
Typically we wrap all of the main code into a `main()` method and then write at the bottom of the file

    if name == "__main__":
        main()

This way, if someone in the future wants to import your script as a module, the import statement won't suddenly start running code.

You can look at some of the other Python scripts here or the stuff in the `utils` repository to see examples.